source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,284 @@
+.. _swupd-3rd-party:
+
+Create, Install, and Manage 3rd-party Bundles
+#############################################
+
+Upstream Clear Linux offers a plethora of bundles to choose from.  
+As of version 32570, there are over 1000 bundles and counting.  
+
+To provide you access to even more software outside of the distro, we have 
+enabled two new features to our :command:`mixer` and :command:`swupd` client tools
+starting with release 32570.  
+
+:command:`mixer` can now create 3rd-party bundles out of any content you want 
+without the need to package it into an RPM format first.  
```

Comment:
I would avoid using the term RPM here to avoid confusion about CL using RPMS. Or if you do, place it in context of mixes/dependency resolution.